ignition/service/resourcedriver.py

```diff
@@ -192,6 +193,7 @@ def __init__(self, **kwargs):
         if 'service' not in kwargs:
             raise ValueError('No service instance provided')
         self.service = kwargs.get('service')
+        self.version = REQUEST_MESSAGE_VERSION
```

Comment:
I don't understand why the version is being set here and passed in to execute_lifecycle below, unless it needs to be configurable here? Shouldn't the request message version be handled only by the request queue message send logic?

Reply:
Addressed in commit 5.